### ISSUE TITLE
plugin Failed to install

### ISSUE THREAD

**Issue by Gev-Tari** [sc0ttkclark/jquery-conditions#2]
Hi;

Following your guys advise I tried to install the plugin and I got the error message "The package could not be installed. No valid plugins were found.Plugin install failed.". There is not information provided if this is a plugin or it should be installed following a particular instruction, so I appreciate if you guys can advise me accordingly.

Regards
Gev

**Comment by Gev-Tari** [comment]
Hi;

I have already created this on your Github and I mentioned this on my last
email.

Regards
Gev

**Comment by sc0ttkclark** [comment]
jQuery Conditions isn't a WordPress plugin, it's a Javascript file you can include within your theme or plugin.

**Comment by Gev-Tari** [comment]
Hi;

Thank you for your reply. It is not really clear to me how it should be
used and I could not find any documentation how to use the .JS and .PHP
file. Can you point me to any documentation available for it?

Regards
Gev

**Comment by sc0ttkclark** [comment]
There's no documentation for it yet, we're still developing it. Targeting it for integration within Pods to help complete it and resolve remaining issues. It would require someone knowledgable in JS / jQuery to use it at this point.

**Comment by Gev-Tari** [comment]
This is the respond from Ninja:

 _James_ replied
Aug 29 11:16am
I'm sorry. That was a mistake on our part. Thus code and repo is not a part
of our company and we have no connection to it. We apologize.

James Laws
The Ninja Forms Team

**Comment by sc0ttkclark** [comment]
Yeah, we're hoping we can work together with different form plugin authors to build a solution that's reusable for the use-cases we all support.
